addOns/oast/src/main/javahelp/org/zaproxy/addon/oast/resources/help/contents/oast.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<html lang="en">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>
+        OAST Support
+    </title>
+</head>
+<body>
+<h1>OAST Support</h1>
+
+The OAST Support add-on allows you to detect and exploit out-of-band vulnerabilities in web applications. <br>
+<br>
+OAST stands for Out-of-band Application Security Testing.
```

Comment:
Should we use the full name in the title and H1?

Reply:
Works for me. If everyone agrees I can update it.

Reply:
Sounds good to me.